frontend-new/src/feedback/feedbackForm/components/yesNoQuestion/YesNoQuestion.tsx

```diff
@@ -63,36 +67,30 @@ const YesNoQuestion: React.FC<YesNoQuestionProps> = ({
     >
       <FormLabel
         component="legend"
-        sx={{
-          fontSize: (theme) => theme.typography.subtitle1.fontSize,
-          "&.Mui-focused": {
-            color: (theme) => theme.palette.common.black,
-          },
-        }}
         data-testid={DATA_TEST_ID.FORM_LABEL}
       >
-        {questionText}
+        <QuestionText questionText={questionText} />
       </FormLabel>
       <RadioGroup
         aria-label={questionId}
         name={questionId}
         value={ratingValue === null ? "" : ratingValue}
         onChange={handleChange}
         row
+        sx={{
+          display: "flex",
+          gap: theme.tabiyaSpacing.lg,
+          marginTop: theme.tabiyaSpacing.md,
+        }}
       >
-        <FormControlLabel value="false" control={<Radio />} label="No" data-testid={DATA_TEST_ID.RADIO_NO} />
-        <FormControlLabel value="true" control={<Radio />} label="Yes" data-testid={DATA_TEST_ID.RADIO_YES} />
+        <FormControlLabel value="false" control={<Radio sx={{ padding: 0 }} />} label="No" data-testid={DATA_TEST_ID.RADIO_NO} sx={{ margin: 0 }} />
```

Comment:
I think we should only remove the paddingY here, the padding x should still be there, otherwise the buttons look too cramped to me...

Reply:
or perhaps we need to explicitly add some horizontal padding